olive/engine/footprint.py

```diff
@@ -105,22 +105,34 @@ def mark_pareto_frontier(self):
         if self.is_marked_pareto_frontier:
             return
         for k, v in self.nodes.items():
+            # unmark all pareto frontier points to be safe
```

Comment:
Maybe no need to set false again?
![image](https://user-images.githubusercontent.com/13343117/231365698-030f1481-98f1-457e-a84b-2eea827e22a3.png)

Reply:
I have removed this part. I think I had some issues with my local testing where the saved json (probably from an old pareto frontier) had some values already marked. Just checked and it is not needed for the correct json.